lib/twitter_cldr/js/mustache/tokenizers/pattern_tokenizer.coffee

```diff
@@ -0,0 +1,24 @@
+# Copyright 2012 Twitter, Inc
+# http://www.apache.org/licenses/LICENSE-2.0
+
+class TwitterCldr.PatternTokenizer
+  constructor : (@data_reader, @tokenizer) ->
+
+  tokenize : (pattern) ->
+    @tokenizer.tokenize(@expand(pattern))
+
+  expand : (pattern) ->
+    # if typeof(pattern) is "string"
+    #   path = (pattern + "").split(".")
+    #   data = (if @data_reader? then @data_reader.pattern_at_path(path) else null)
+    #   next_pattern = (if data["pattern"]? then data["pattern"] else data)
+    #   expand_pattern(next_pattern)
+    # else
+    # TODO - figure thsi out.
```

Comment:
I don't think this is used right now btw. All tests for all locales (except the ones with allowed failures) pass right now. I'm not sure if this is required in the first place. I'll investigate this and add it in anyway.